ci: MSVC failures

The Windows native CI is failing on unrelated changes, i.e in #29072.
https://github.com/bitcoin/bitcoin/actions/runs/7194911390/job/19596459746?pr=29072:
```bash
rpc_help.py                                            | ✖ Failed  | 1 s
rpc_signer.py                                          | ✖ Failed  | 4 s
wallet_signer.py --descriptors                         | ✖ Failed  | 3 s
```

```bash
  node0 2023-12-13T12:48:04.392953Z [D:\a\bitcoin\bitcoin\src\rpc\request.cpp:187] [parse] [rpc] ThreadRPCServer method=help user=__cookie__ 
 test  2023-12-13T12:48:04.393000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "D:\a\bitcoin\bitcoin\test\functional\test_framework\test_framework.py", line 131, in main
                                       self.run_test()
                                     File "D:\a\bitcoin\bitcoin/test/functional/rpc_help.py", line 55, in run_test
                                       self.test_categories()
                                     File "D:\a\bitcoin\bitcoin/test/functional/rpc_help.py", line 117, in test_categories
                                       assert_equal(titles, sorted(components))
                                     File "D:\a\bitcoin\bitcoin\test\functional\test_framework\util.py", line 57, in assert_equal
                                       raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
                                   AssertionError: not(['Blockchain', 'Control', 'Mining', 'Network', 'Rawtransactions', 'Util', 'Wallet', 'Zmq'] == ['Blockchain', 'Control', 'Mining', 'Network', 'Rawtransactions', 'Signer', 'Util', 'Wallet', 'Zmq'])
 test  2023-12-13T12:48:04.393000Z TestFramework (DEBUG): Closing down network thread 
```

--- a/build_msvc/bitcoind/bitcoind.vcxproj
+++ b/build_msvc/bitcoind/bitcoind.vcxproj
@@ -74,15 +74,19 @@
     <ReplaceInFile FilePath="$(ConfigIniOut)"
                   Replace="@BUILD_BITCOIN_CLI_TRUE@" By=""></ReplaceInFile>
     <ReplaceInFile FilePath="$(ConfigIniOut)"
+                  Replace="@BUILD_BITCOIN_UTIL_TRUE@" By=""></ReplaceInFile>
+    <ReplaceInFile FilePath="$(ConfigIniOut)"
                   Replace="@BUILD_BITCOIN_WALLET_TRUE@" By=""></ReplaceInFile>
     <ReplaceInFile FilePath="$(ConfigIniOut)"
                   Replace="@BUILD_BITCOIND_TRUE@" By=""></ReplaceInFile>
     <ReplaceInFile FilePath="$(ConfigIniOut)"
-                  Replace="@ENABLE_FUZZ_TRUE@" By=""></ReplaceInFile>
+                  Replace="@ENABLE_FUZZ_BINARY_TRUE@" By="#"></ReplaceInFile>
     <ReplaceInFile FilePath="$(ConfigIniOut)"
                   Replace="@ENABLE_ZMQ_TRUE@" By=""></ReplaceInFile>
     <ReplaceInFile FilePath="$(ConfigIniOut)"
-                  Replace="@ENABLE_EXTERNAL_SIGNER_TRUE@" By=""></ReplaceInFile>
+                  Replace="@ENABLE_EXTERNAL_SIGNER_TRUE@" By="#"></ReplaceInFile>
+    <ReplaceInFile FilePath="$(ConfigIniOut)"
+                  Replace="@ENABLE_USDT_TRACEPOINTS_TRUE@" By="#"></ReplaceInFile>
   </Target>
   <Import Project="..\common.vcxproj" />
 </Project>

